src/slashCommand/commands/randomcabin.command.ts

```diff
@@ -17,4 +17,7 @@ export default class RandomCabinCommand {
         .setDescription('When do you want to leave (yyyy-mm-dd)?')
         .setRequired(true),
     );
+
+  // Defer response
+  // https://discordjs.guide/interactions/slash-commands.html#editing-responses
```

Comment:
![image](https://user-images.githubusercontent.com/1177034/196524852-84984d9d-cba1-4856-9bda-0e458a815c7b.png)
This URL doesn't seem to work for me.

Reply:
I literally copied it from the browser, but now it doesn't work for me either (also the link indexed by google gives a 404, so must be something recent). Try this instead: https://v13.discordjs.guide/interactions/slash-commands.html#deferred-responses

Logic needd to be inmplemented once the slash command feature is updated.